src/action/mod.rs

```diff
@@ -243,11 +243,11 @@ impl Action {
                 let mut userpicked = UserPicked { bandaids: indexmap::IndexMap::new() };
                 for (origin, suggestion) in suggestions.iter() {
                     log::trace!("Reflows in file {}:", origin);
-                    let bandaids = suggestion.iter().map(|s| {
-                        log::trace!("{}", s);
+                    let bandaids: Vec<BandAid> = suggestion.iter().map(|s| {
+                        log::trace!("{:?}", s);
                         // @todo how can I use `?` here? rustc complains about missing `std::ops::Try` for BandAid
```

Comment:
you cannot, you have to either use `try_fold` and combine the `map` with the `collect`, but you cannot use `?` in closure which does return something that does not impl `Try`